docs/components/plugins/InstallSection.tsx

```diff
@@ -65,3 +66,8 @@ const InstallSection: React.FC<Props> = ({
 );
 
 export default InstallSection;
+
+export const APIInstallSection: React.FC<Props> = props => {
```

Comment:
Is there a reason why you have added the new export instead reusing old name and change what file is exporting?

Also I'm curious what do you think about dropping `FC` usage, I have bring up this topic earlier in few other PRs and this idea is  gaining some supporters in the team: 
* https://github.com/facebook/create-react-app/pull/8177
* https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/

Reply:
Because this component is unversioned, I only wanted to apply this to the latest version and keep old functionality functioning like before